docs/spec/spec.md

```diff
@@ -0,0 +1,310 @@
+| Update | Provisioner controller synchronizes backing implementation on changes.                                    |             |
+| Delete | Provisioner controller will deprovision backing resources depending on implementation.                    |             |
+
+---
+
+## kind: Channel
+
+### group: eventing.knative.dev/v1alpha1
+
+_A Channel logically receives events on its input domain and forwards them to
+its subscribers. Additional behavior may be introduced by using the
+Subscription's call parameter._
+
+### Object Schema
+
+#### Spec
+
+| Field         | Type                               | Description                                                                | Limitations                          |
+| ------------- | ---------------------------------- | -------------------------------------------------------------------------- | ------------------------------------ |
+| provisioner\* | ProvisionerReference               | The name of the provisioner to create the resources that back the Channel. | Immutable.                           |
+| arguments     | runtime.RawExtension (JSON object) | Arguments to be passed to the provisioner.                                 |                                      |
+| channelable   | Channelable                        | Holds a list of downstream subscribers for the channel.                    |                                      |
+| eventTypes    | []String                           | An array of event types that will be passed on the Channel.                | Must be objects with kind:EventType. |
+
+\*: Required
+
+#### Metadata
+
+##### Owner References
+
+- If the Source controller created this Channel: Owned by the originating
+  Source.
+- Owned (non-controlling) by the Provisioner used to provision the Channel.
+
+#### Status
+
+| Field        | Type         | Description                                                                                                                 | Limitations |
+| ------------ | ------------ | --------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| Sinkable     | Sinkable     | Address to the endpoint as top-level domain that will distribute traffic over the provided targets from inside the cluster. |             |
+| subscribable | Subscribable |                                                                                                                             |             |
+| Conditions   | Conditions   | Standard Subscriptions                                                                                                      |             |
+
+##### Conditions
+
+- **Ready.** True when the Channel is provisioned and ready to accept events.
+- **Provisioned.** True when the Channel has been provisioned by a controller.
+
+#### Events
+
+- Provisioned
+- Deprovisioned
+
+### Life Cycle
```

Comment:
Again, not sure "Life Cycle" as a table is the best way to represent this. Possibly "Implementation notes"?

Reply:
followup.